### PR TITLE
chore(points): Use activityId and pointsAmount throughout

### PR DESCRIPTION
**Pull request by jophish**
### Description

Updates the points implementation + redux to use `activityId` and `pointsAmount` everywhere, as discussed in Slack.

### Test plan

Manual/unit tested.

### Related issues

Part of [ACT-1032](https://linear.app/valora/issue/RET-1032/[wallet-fake-backend]-points-activity-history).

### Backwards compatibility

Yes.

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [x] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: src/points/ActivityCard.tsx [hunk 1]
```diff
@@ -11,26 +11,26 @@ import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
 import { PointsEvents } from 'src/analytics/Events'
 
 interface Props {
-  activity: PointsActivity
+  activityId: PointsActivity
   points: number
```
Comment (kathaypacific) [inline src/points/ActivityCard.tsx:15]: should this also be `pointsAmount`?

**Review by kathaypacific — APPROVED** [review]
💅🏼🚀

File: src/points/ActivityCard.tsx [hunk 2]
```diff
@@ -11,26 +11,26 @@ import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
 import { PointsEvents } from 'src/analytics/Events'
 
 interface Props {
-  activity: PointsActivity
+  activityId: PointsActivity
```
Comment (bakoushin) [inline src/points/ActivityCard.tsx:14]: should we change the type name to `PointsActivityId` as well?

File: src/points/ActivityCard.tsx (hunk 1 above)
Comment (jophish) [reply src/points/ActivityCard.tsx:15]: yep, good call!

**Review by kathaypacific — APPROVED** [review]

**Pull request merged**